sample-apps/aws-xray-sdk-go-sample/main.go

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
change the headers ? now it should be apache.

Reply:
Added

Reply:
Sure